extensions/amp-animation/0.1/web-animations.js

```diff
@@ -0,0 +1,363 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {dev, user} from '../../../src/log';
+import {getVendorJsPropertyName} from '../../../src/style';
+import {isArray, isObject} from '../../../src/types';
+import {
+  WebAnimationDef,
```

Comment:
interesting way of doing this. Does it make sense to move the type defs to amp.extern?